plugins/modules/msk_cluster.py

```diff
@@ -54,6 +54,10 @@
             - kafka.m5.xlarge
             - kafka.m5.2xlarge
             - kafka.m5.4xlarge
```

Comment:
Can you please add a note like "Support for C(kafka.m5.8xlarge), C(kafka.m5.12xlarge), C(kafka.m5.16xlarge), and C(kafka.m5.24xlarge) was added in version 7.1.0"?

Reply:
Thank you for the review. I will follow up right away.

Reply:
The latest MSK Broker also supports m7g type. I will update the changelog after adding it.

Reply:
I added the change log fragments, and I added the m7g type because it recently became available. Thanks for the feedback.